src/pages/SignIn/SignIn.js

```diff
@@ -1,7 +1,98 @@
 import React from 'react';
+import styled from 'styled-components';
+// import KakaoLogin from 'react-kakao-login';
 
-function SignIn() {
-  return <div>SignIn</div>;
-}
+const SignIn = () => {
+  return (
+    <Wrapper>
+      <SignUpBox>
+        <Title>로그인</Title>
+        <div>
+          <InputBox placeholder="이메일 주소" type="text" />
+          <InputBox placeholder="비밀번호" type="text" />
+        </div>
+        <div>
+          <Label>
+            <div>
+              <input type="checkbox" />
+              로그인 상태 유지
+            </div>
+          </Label>
+        </div>
+        <ButtonWrapper>
+          <div>
+            <button className="emailBtn">로그인</button>
+          </div>
+          <div>
+            <button className="kakaoBtn">카카오로 로그인</button>
+          </div>
+        </ButtonWrapper>
+      </SignUpBox>
+    </Wrapper>
+  );
+};
 
 export default SignIn;
+
+const Wrapper = styled.div`
+  ${props => props.theme.flexMixin('center', 'center')};
```

Comment:
```suggestion
  ${({ theme }) => theme.flexMixin('center', 'center')};
```
이렇게 구조분해 할당 해줄 수 있습니다~

Reply:
오오 감사합니다.